File: Two Number Sum/twoNumberSum.js
function twoNumberSum(array, targetSum) {
  for (let i = 0; i < array.length; i++) {
    for (let j = i + 1; j < array.length; j++) {
      if (targetSum === array[i] + array[j]) {
        return [array[i], array[j]];
      }
    }
  }
  return [];
}

exports.twoNumberSum = twoNumberSum;

File: Two Number Sum/twoNumberSum.cpp
#include <vector>
using namespace std;

vector<int> twoNumberSum(vector<int> array, int targetSum)
{
    for (int i = 0; i < array.size(); i++)
    {
        for (int j = i + 1; j < array.size(); j++)
        {
            if (targetSum == array[i] + array[j])
            {
                return vector<int>{array[i], array[j]};
            }
        }
    }
    return {};
}